### ISSUE TITLE
[Bug] fetch_order_book on BL3P exchange

### ISSUE THREAD

**Issue by yoursbh** [kroitor/ccxt#213]
Hello,

I got a key error when I try to get the 'LTC/EUR' orderbook over BL3P exchange. Here are the codes and error message:
```
def main():
    test = ccxt.bl3p()
    market = test.load_markets()
    print market
    print test.fetch_order_book('LTC/EUR')
```

> {'BTC/EUR': {'quote': 'EUR', 'symbol': 'BTC/EUR', 'base': 'BTC', 'id': 'BTCEUR'}, 'LTC/EUR': {'quote': 'EUR', 'symbol': 'LTC/EUR', 'base': 'LTC', 'id': 'LTCEUR'}}
> Traceback (most recent call last):
>   File "arbitrage.py", line 248, in <module>
>     main()
>   File "arbitrage.py", line 209, in main
>     print test.fetch_order_book('LTC/EUR')
>   File "/usr/local/lib/python2.7/site-packages/ccxt/exchanges.py", line 4928, in fetch_order_book
>     return self.parse_order_book(orderbook)
>   File "/usr/local/lib/python2.7/site-packages/ccxt/exchange.py", line 677, in parse_order_book
>     'bids': self.parse_bidasks(orderbook[bids_key], price_key, amount_key),
> KeyError: 'bids'

**Comment by kroitor** [comment]
Hi! We have uploaded the fix for this, but it will take some time for a build to pass on Travis, will be ready in a few minutes. Thx for reporting!

**Comment by kroitor** [comment]
This bug was fixed as of version 1.7.46+. Please update and check if the problem is gone now. We tested it and it should work normally. We will be happy if you report back whether it works or not. Let us know if you have any difficulty with it.
